apps/components-e2e/src/components/filter-field/filter-field.e2e.ts

```diff
@@ -76,3 +83,145 @@ test('should remove all filters when clicking the clear-all button', async (test
   await testController.wait(300);
   await testController.expect(await filterTags.exists).notOk();
 });
+
+test('should choose a freetext node with the keyboard and submit the correct value', async (testController: TestController) => {
```

Comment:
use the waitForAngular inside a `fixture.beforeEach(async () => waitForAngular())`

and please try to chain everything that returns a `TestControllerSelectorPromise` Furthermore inside the `expect(...)` no await is needed the expect can handle Promises.

This increases the stability of tests a lot